src/modules/menu/components/contentPanel/contentPanel.css

```diff
@@ -181,6 +181,7 @@
ZCehqs0jPuBrHmKYCmnEjtVHvVvFjl9z4Ixu8ye2KKRZ5BaZNnU22sXC2uAWZJlZO50gink6Kmq7l/F9U4KST+1fxwM0iwhm/KWWXQkdKjFZMjSRdvyITBLHNHG5yt27kRNfb6cTkuf2m+MZ0qacL5sqVPytTzTE3HRsonyJhHry2Inrqx29WJ7d2obYP7MPE/WiadRMIbMliOa0Wh3dC/JVgKYTHJoqjVzdVVF1VNTZFhHigWJ2dnGi19lIxnPfKFN3WHBEgKyPcQ67N4sS2jRgsaOP1DCNGxEa3REROHW4sJrUtI5Elus1lW5J6yHVMalSSSdJsHFlSVpoQornPe96xxNYvoiIoxt8E+IIsaP1PKFDwunhRwueZkiXLRoYgRjLIkF1LJVN7le7c9dV1c6v8X4LCMUBRPFAhiCx0KSZheUYKEaJ8WSeOjkY9qse9uiIq8WeL4vjGExH5lNv493jGMvlinXU9XyxZLBtoKTgyBPd203VCNyxKgtXJoiKkSJJwfG5sbEqx0DExznWEkkioxXH4eQBgV3UldOJSUGM3kaSxhVLGidSjERDInBgeKf28RULNI7IXT4+HSK+vNNFHOASktZMhDV91PABAgRzmKxj2Ofo1NOLSp8B/s9x3Np1HcxqZ0WpqLDLzY7MbK7eZcgjxRAxvHwwRi1esqaDksRWbFc3bxMqs08ZftKwmurpZpIsPDizPKdkwKyAmJMtqqlixqGvsCxEVHiWzmv37XuREXRKbKm4BhoYEYsmGXEm+OMcZgkuuPUzK8LFwMFU6nMYQGvK7nhaRFaN7XqRGcTrOpI62nVsR70hukQIgZToZYMOQJXOseUNqdbqxpDoiCXehHaJrl4zWMioZSyY1lBkGUhQBFPppMpko0OA9VjwLZRSVaWQvJYQatRGkVjFtrqbeRyRoVGyxF1jlSXYTQWkeuqYDgiOWIKVLFOY/krJ57DR1arHaO08eeO8ls7akv8AP/H935RpSFqbFQTcZpbOwxO4B3YYpECLmNTOpGx1hMaqBM3eRE13cAtAXc+HAr7EsLJgyYMKRHWCSrG5gAChT1kNICVKCZTE9EchWojXN2IrMeES2rmRpToM2j6pWJYcyBHnSISKwcqyp3sr0DIcHYWMKcVyFRRe3DpUGjvL+ZFjZXXZFDmxJQpeQWCAOkYAYUk9Q2uaCOqyCpJkEfsE5FI9VRrPEgZecvfeR/B3i+Rkra2e+AWyn3HkPPsttawdSqnYSLSVs5OUFXOOIzWKxXPXRvknz6QrgTcn8keUMipLWxmV4KiBW5NkWU08GHXQIbCyGADQ2IyNbIE1zHI32NRj2tmXFjmtrBrKHGu1UdBjttApot7AHUQaWKlDRRQjkyTDjVwYo/aiSiBa3VXJqlfmv7mj30bHIMt95gnjGNY2NJkVpU2iMK6P587eIUS9PJa1rn0wTGjiI5/MMRXuakyjxaursdrotdPbX1VJUwaeujG6Mz1eGHWxRCYZ701VyDV7n+q7l+pRZHNO0xZk20ikR9upIwSywnc/QMcUkZ2EVW7HK5Hi0T6aokKUzIJK1Um0WqNaEFOGxalIE6yQQYii5R5kcEZippHRzQyCKur9Ht8i5tf2MXBY4ayZSY9iEqTX0tq6zJUnsZ12WTf1Marh1Migr1Qkt5RdY4kfZHRd7+K/ORGt7c1TiOPvynFbabSrCva+qJayKmyHVxVkGsoUSAASODIfJ2nVVc8fs3koaUeN2EAYa/HXwZ+E1JZBMUxy1jpbSDyq55XXLlM18grpcsra9sVhQSCv5yi/bz5AkZbd2WQeKcr8/eObAE403IOmwiwl4Z5FoBuiJspilHkmdWUWSg3/AHRNY3RzWtdxltLcXlvObj9SWeeFKggiWdm7IolLz4bZ0iytRT5WHsixiRgneBp1PLR2iOaiVljlC2ErJ4GOUMCHZQSgniBX10oRA3UKkYSXOo3S4xjkHBYjSSeYrlG1io5Txsa8dTMbxCHTDNW3Rp0Md0p7i3K0aSY8pSKJ1nWSnyny+YqsEVw3at5jOPHmc5NlttSVcSvgjsq3HnGgmLRAmPbXXE5pDR5m+Y06lcjhKKExFcNWOLq+mh4pfBn+Mphpl2aTYFCyH2GpkGtngjTmFmQj2MsrNo0OoXE/E8ikUiOofJmeYxNpME8fneLx912O3N0K4lDkkKO7iTI8O1oLWJVgIMYkKQsVxdz0ZuT0hxyyHSihCgyyngjwuocNNvOdFiABEFr/AKRDYxP4InARFVhBvVRma0iI7mPa5E9yN9HORdVVfTT+PB3Wj4badVlS6ZTpzUNWNf17COOFhJLyAGrUJojdHfRFVFcg4gmwukjFZJkueaG5vTnZLVrgOcTVpOSMfMa1+iE2tV216KrfmXe+R1NL37571nR9J2u5/wDpOf7ui6f8XUe3tnM0+xv4sux9R0nQs7jp+Lm9vr+0/COq/uPbup5PWa/mdOb/AA04q+m5f4287451HyDpO+w+x9J1v2O3but6jT39Du6j2cX36ld/+I/qXj/xrXldb8c24v8AqZyuh/N9/wDj/Q8/q/Xqf9v7G3jCu0dN2noV/wDZ8vsHeN6dB8l6z2dRr0+3m+vP5nM+xv4tvjHUc3rKv4X3bunYu6dIXrPkfd/7js6npOq/ldJy+V9vfwTt/wAh6/5XQd15XT/Dev7q7l9R1H3+19Fs7bzvZy9n8zdxV/I+6/FtJ/U95+Kdi5HMxj4h3rT+zdq7V1PL7d7eZpz/AHczix7r2v8AQj57H29L036gfAfktd3js/ZPy3e+m6jTd+U6jn/02zjDf0q7X+nfxuq+EfGu2dm7H0AOg7d2n+z6crbu5f8AM14P+DfsJ/vbNfqn+j/x+v8Anprwzlcrl8tOZzuXv27n8r6+3X6fT1+mvGLd/wDjvce/Re1d+7Z03bd7+t53M/M8vqOXs5P2+bv3+mvEnb2PoO4e/Tt/L6LlRNOZ/S/He3acr+o6fm/9/H//2Q==');;
```

Comment:
For now, we don't want to include any raster images. Would you revert this change, please?